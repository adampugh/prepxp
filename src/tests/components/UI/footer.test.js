import React from "react";
import { shallow } from "enzyme";

import Footer from "../../../components/UI/footer";

describe("<Footer />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });
});