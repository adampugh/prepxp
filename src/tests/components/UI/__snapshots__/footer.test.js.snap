// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
<footer>
  <div
    className="container"
  >
    <div
      className="footer__grid"
    >
      <div
        className="footer__grid__panel"
      >
        <h1
          className="logo"
        >
          PrepXP
        </h1>
        <p>
          <i
            className="fab fa-twitter"
          />
          <i
            className="fab fa-facebook-f"
          />
          <i
            className="fab fa-youtube"
          />
        </p>
        <p>
          info@prepxp.com
        </p>
      </div>
      <div
        className="footer__grid__panel"
      >
        <h2>
          Explore
        </h2>
        <p
          className="footer__grid__panel__link"
        >
          •  Home
        </p>
        <p
          className="footer__grid__panel__link"
        >
          •  About
        </p>
        <p
          className="footer__grid__panel__link"
        >
          •  Blog
        </p>
      </div>
      <div
        className="footer__grid__panel"
      >
        <h2>
          Latest Blog Posts
        </h2>
        <div
          className="footer__grid__blogPost"
        >
          <p>
            5 Most common interview questions
          </p>
        </div>
        <div
          className="footer__grid__blogPost"
        >
          <p>
            How to pass the technical interview
          </p>
        </div>
      </div>
    </div>
    <p
      className="footer__tagline"
    >
      © 2017 Design By 
      <a
        href="https://adampugh.github.io/kokonoka/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Kokonoka Digital
      </a>
    </p>
  </div>
</footer>
`;
